extensions/script-kit/src/run-script.tsx

```diff
@@ -0,0 +1,97 @@
+import { ActionPanel, closeMainWindow, Icon, List, popToRoot } from "@raycast/api";
+import { spawn } from "child_process";
+import { readFile } from "fs/promises";
+import { homedir } from "os";
+import { resolve } from "path";
+import { useEffect, useState } from "react";
+import { Script } from "./script";
+
+const kit_path = resolve(homedir(), ".kit");
+
+export default function listScripts(): JSX.Element {
+  const [scripts, setScripts] = useState<Script[]>();
+  useEffect(() => {
+    const script_db_path = resolve(kit_path, "db", "scripts.json");
+    readFile(script_db_path)
+      .then((buffer) => buffer.toString())
+      .then(JSON.parse)
+      .then((json) => setScripts(json.scripts));
+  }, []);
+
+  return (
+    <List>
+      {scripts
+        ?.filter((script) => !script.exclude)
+        .map((script) => (
+          <List.Item
+            title={script.name}
+            key={script.command}
+            subtitle={script.description}
+            accessoryTitle={script.kenv}
+            actions={
+              <ActionPanel>
+                <ActionPanel.Section>
+                  <RunScriptAction scriptPath={script.filePath} />
+                </ActionPanel.Section>
+                <ActionPanel.Section>
+                  <NewScriptAction/>
+                  <NewScriptFromUrlAction/>
+                </ActionPanel.Section>
+                <ActionPanel.Section>
+                  <ShareScriptAsGistAction scriptPath={script.filePath}/>
+                  <EditScriptAction scriptPath={script.filePath}/>
+                  <DeleteScriptAction scriptPath={script.filePath}/>
+                </ActionPanel.Section>
+              </ActionPanel>
+            }
+          />
+        ))}
+    </List>
+  );
+}
+
+function DeleteScriptAction(props: { scriptPath: string }) {
+  return (
+    <RunScriptAction
+      title="Remove Script"
+      icon={Icon.Trash}
+      scriptPath={resolve(kit_path, "cli", "remove.js")}
+      args={[props.scriptPath]}
+    />
+  );
+}
+
+function NewScriptAction() {
+  return <RunScriptAction title="New Script" icon={Icon.Plus} scriptPath={resolve(kit_path, "cli", "new.js")} />;
```

Comment:
Sometime in the future, I might provide a `clis.json` you could generate these Actions from. 🤔